hw/dv/tools/vcs/common_cov_excl.cfg

```diff
@@ -13,6 +13,8 @@
 -node tb.dut *tl_o.d_sink
 // [UNR] due to the ECC logics
 -node tb.dut *tl_o.d_user.rsp_intg[6]
+// [UNR] unused inputs at prim_secded_* in reg_top and mem.
+-node tb.dut*_chk.u_chk data_i[56:43]
```

Comment:
great, thanks! I saw this in the generated UNR for alert_handler as well.

Reply:
Thanks Cindy for checking this.